components/atom/tag/src/index.js

```diff
@@ -96,6 +104,19 @@ AtomTag.propTypes = {
    * To be used if href is defined
    */
   target: PropTypes.oneOf(['_self', '_blank', '_parent', '_top']),
+  /**
+   * To be used if href is defined
+   */
+  rel: PropTypes.arrayOf(
+    PropTypes.oneOf([
```

Comment:
replace by: `PropTypes.oneOf(Object.values(LINK_TYPES))` so there is no need to modify this line if you add more LINK_TYPES in the future.